iceoryx_utils/include/iceoryx_utils/internal/posix_wrapper/unix_domain_socket.hpp

```diff
@@ -35,65 +32,87 @@ namespace posix
 class UnixDomainSocket : public DesignPattern::Creation<UnixDomainSocket, IpcChannelError>
 {
   public:
-    /// @ todo in ipc_channel.hpp? The same for all channels?
     static constexpr size_t MAX_MESSAGE_SIZE = 4096;
+    static constexpr size_t SHORTEST_VALID_NAME = 2;
```

Comment:
std::size_t is unsigned integer type therefore we need to add an "u"

Reply:
done